LandSandBoat/server
# Lycopodium Mixin; Granite Door requires Uggalepih Key

**_I affirm:_**

## What does this pull request do?

- Granite Door at E-8 in Temple of Uggalepih now requires an Uggalepih Key
- Lycopodiums in the past will now follow players for a short time similarly to Hpemdes in Al'teaiu

## Steps to test these changes
- Door: Attempt to open the door from both sides, ensure it opens from insde the room and not from outside.
- Lycopodiums: With GM flag turned off, move around Lycopodiums and see that they follow you around for a short time.
